lib/pure/parseutils.nim

```diff
@@ -211,6 +211,7 @@ proc parseHex*[T: SomeInteger](s: string, number: var T, start = 0,
 proc parseIdent*(s: string, ident: var string, start = 0): int =
   ## Parses an identifier and stores it in ``ident``. Returns
   ## the number of the parsed characters or 0 in case of an error.
+  ## If error, the value of ``ident`` is not changed.
```

Comment:
Maybe use `ident` which is more simple.

Reply:
I can definitely do that, but then I would change it throughout the document (lots of other places where double backticks appear) to keep it consistent. should I do that?

Reply:
Yeah, two ways exists. You don't have to clean them in this PR, just not introducing them is enough.
See https://github.com/timotheecour/Nim/issues/415

Reply:
done